python/tests/test_grounded_type.py

```diff
@@ -24,9 +24,6 @@ def test_apply_type(self):
         self.assertEqual(
             metta.interpret("(untop)")[0].get_grounded_type(),
             metta.parse_single("untyped").get_grounded_type())
-        self.assertNotEqual(
```

Comment:
Maybe, I would not remove this, but just comment out. While it is clear that having an untyped operation of the type `(-> Undefined)` is incorrect, because it is arity will be wrong, but we still may want to distinguish types of operations and non-operational atoms. We discussed that it would be nice to have something like `*args` in the future. The type of untyped operations is yet another example showing this can be useful. So, the type of untyped operations might become something like `(-> *Undefined Undefined)` in the future. Of course, if this happens, we can add this unit test later, so it is not actually necessary to return it in the commented form.

Reply:
Yes, I also hesitated to remove it. And main reason is to not forget add check for this. I can add separate unit test for it and ignore it with TODO comment. Then it will be more visible as ignored unit test rather then commented out part of the test.

Reply:
Sounds good

Reply:
Fixed in #99